File: 백준/Bronze/10886. 0 ＝ not cute ／ 1 ＝ cute/0 ＝ not cute ／ 1 ＝ cute.py
n = int(input())
yes, no = 0, 0
 
for i in range(0, n):
    num = int(input())
    if num == 1:
        yes += 1
    else:
        no += 1
if yes > no:
    print("Junhee is cute!")
else:
    print("Junhee is not cute!")

File: 백준/Bronze/10886. 0 ＝ not cute ／ 1 ＝ cute/0 ＝ not cute ／ 1 ＝ cute.cc
#include <stdio.h>

int main(){
    int n, num, yes=0, no=0;
    scanf("%d", &n);
    for(int i=0; i < n; i++){
        scanf("%d", &num);
        if(num == 1){
            yes += 1;
        }else{no += 1;}
    }
    if(yes > no){printf("Junhee is cute!");}
    else if(yes < no){printf("Junhee is not cute!");}
    return 0;
}